src/Aspire.Dashboard/Components/App.razor

```diff
@@ -39,3 +42,22 @@
     <script src="_content/Microsoft.FluentUI.AspNetCore.Components/Microsoft.FluentUI.AspNetCore.Components.lib.module.js" type="module"></script>
 </body>
 </html>
+
+@code {
```

Comment:
Move this to a code behind file. Refactoring and code analyzers work better with code behind cs vs cs in the razor file.